### ISSUE TITLE
Decommission lib-sc-staging[12].princeton.edu

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/princeton_ansible#5407]
### What maintenance needs to be done?

Decommission lib-sc-staging[12].princeton.edu

### Why is this maintenance needed?

These vms are no longer needed, since we no longer host and maintain the drupal library website.

### Acceptance criteria
 - [ ] These VMs are not referenced in load balancer configurations
 - [ ] These VMs are not in the inventory
 - [ ] delete lib-sc-staging1.princeton.edu vm
 - [ ] delete lib-sc-staging2.princeton.edu vm
 - [ ] delete DNS for the VMs
 - [ ] delete firewall rules

**Comment by dphillips-39** [comment]
Both VM's were disconnected from the network and will be powered off on Oct 10, 2024

**Comment by dphillips-39** [comment]
This is blocked by https://github.com/pulibrary/princeton_ansible/issues/5531

**Comment by dphillips-39** [comment]
Blocked by https://github.com/pulibrary/pul_library_drupal/issues/2072

**Comment by VickieKarasic** [comment]
@dphillips-39 We are aiming to power these down on 12/19/24 for 30 days and then fully decommission in the new year.

**Comment by dphillips-39** [comment]
VM's disconnected from server, to be fully decommissioned in January 2025.
